packages/react-server/core/context/Navigator.js

```diff
@@ -50,7 +50,7 @@ class Navigator extends EventEmitter {
 
 		this._haveInitialized = true;
 
-		var route = this.router.getRoute(request.getUrl(), {navigate: {path:request.getUrl(), type:type}});
+		var route = this.router.getRoute(request.getUrl(), {method: request.getMethod()});
```

Comment:
I found myself wondering: how come we don't need `type` in here anymore? And then I found myself wondering: why did we need it in the first place?

The only thing I can think of offhand that might break here, is if `type` is being stored on the `route` object that's returned and we're reaching into it instead of using the `type` we're passing around everywhere.

I looked at [the commit](https://github.com/yahoo/routr/pull/29/files) in `routr`, and did some grepping, and I think we're probably in the clear. 👍